### PR TITLE
Add version 1.15 - move interfaces into them

### PR DESCRIPTION
**Pull request by dylant-da**
Simplified version of https://github.com/digital-asset/daml/pull/14720

Guards handled in https://github.com/digital-asset/daml/pull/14773

### CODE REVIEW

File: daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala [hunk 1]
```diff
@@ -74,11 +74,11 @@ object LanguageVersion {
   // All the stable and preview versions
   // Equals `Stable` if no preview version is available
   val EarlyAccessVersions: VersionRange[LanguageVersion] =
-    StableVersions
+    StableVersions.copy(max = v1_15)
```
Comment (remyhaemmerle-da) [inline daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala:77]: We should not make 15 stable.

Comment (dylant-da) [reply daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala:77]: We're defining Early Access in terms of Stable, adding 1.15 to the list of early access versions - we don't modify Stable here. Or does .copy also mutate the original VersionRange?

Comment (remyhaemmerle-da) [reply daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala:77]: my bad.

**Review by remyhaemmerle-da — APPROVED** [review]
LGTM. 

I would appreciate if @akrmn took a closest look at Haskell side.

File: ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15.scala [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.tls.TlsConfiguration
+
+package object v1_15 {
```
Comment (skisel-da) [inline ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15.scala:9]: What is actually a relationship between 1.15 and 1.dev currently? Why InterfaceSubscriptionsIT is not part of 1.15?

Comment (remyhaemmerle-da) [reply ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15.scala:9]: We will add back in ahtoner PR.

Comment (remyhaemmerle-da) [reply ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15.scala:9]: Will add it in the next PR.

**Pull request merged**